resources/views/devices/group-list.blade.php

```diff
@@ -0,0 +1,49 @@
+@extends('layouts.app')
+
+@include('includes.datatables')
+
+@section('pagecss')
+    @parent
+    <link href="{{ url('js/plugins/querybuilder/query-builder.default.min.css') }}" rel="stylesheet"/>
+    <link href="{{ url('js/plugins/select2/select2.min.css') }}" rel="stylesheet"/>
+
+@endsection
+
+@section('pagejs')
+    @parent
+    <script src="{{ url('js/core/modals.js') }}"></script>
+    <script src="{{ url('js/plugins/querybuilder/query-builder.standalone.js') }}"></script>
```

Comment:
Should be:

<script src="{{ url('js/plugins/querybuilder/query-builder.standalone.min.js') }}"></script>